### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by manasdk**
[Sorta like assorted candy and nuts but worse]
- There are some bonehead bugs in our workflows. Trying to avoid whatever snafus possible.

### CODE REVIEW

File: actions/st2_prepare_for_release.meta.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   runner_type: "action-chain"
   description: "Look at workflows/st2_prepare_for_release.yaml"
   enabled: true
-  entry_point: "workflows/st2_pytests_setup.yaml"
+  entry_point: "workflows/st2_prepare_for_release.yaml"
```
Comment (lakshmi-kannan) [inline actions/st2_prepare_for_release.meta.yaml:6]: Ugh. Not sure how this got messed up. Was it me?

File: actions/workflows/st2_package_publish_release.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    -
+      name: "setup_prod_pkg_rule_for_branch"
+      ref: "st2cd.create_prod_pkg_rule"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        st2-base-url: "{{st2_base_url}}"
+        branch: "v{{st2_release_base_ver}}"
+      on-success: "run_ubuntu_prod_pkg_for_release"
+    -
+      name: "run_ubuntu_prod_pkg_for_release"
```
Comment (lakshmi-kannan) [inline actions/workflows/st2_package_publish_release.yaml:45]: Won't this kick off the packaging process again? Just wanna make sure you want these as part of this chain.

File: actions/workflows/st2_package_publish_release.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      ref: "core.local"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        st2-base-url: "{{st2_base_url}}"
+        branch: "v{{st2_release_base_ver}}"
+        cmd: "st2 run st2_pkg_ubuntu14 repo={{st2_repo}} branch=v{{st2_release_base_ver}} dl_server={{system.apt_origin_production}} environment=production revision={{st2_revision}} build={{build_key}} -a"
```
Comment (DoriftoShoes) [inline actions/workflows/st2_package_publish_release.yaml:51]: Why do we use core.locals here?  Shouldn't we just natively call the action?

File: actions/st2_package_publish_release.meta.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+  name: "package-publish-release"
```
Comment (lakshmi-kannan) [inline actions/st2_package_publish_release.meta.yaml:2]: s/-/_

File: actions/workflows/st2_package_publish_release.yaml (hunk 3 above)
Comment (lakshmi-kannan) [inline actions/workflows/st2_package_publish_release.yaml:51]: @manasdk says async bro

File: actions/st2_prepare_for_release.meta.yaml (hunk 1 above)
Comment (manasdk) [inline actions/st2_prepare_for_release.meta.yaml:6]: If the shoe fits :)

File: actions/workflows/st2_package_publish_release.yaml (hunk 2 above)
Comment (manasdk) [inline actions/workflows/st2_package_publish_release.yaml:45]: Yes!

File: actions/workflows/st2_package_publish_release.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      on-success: "slack_success"
+      on-failure: "slack_failure"
+    -
+      name: "slack_success"
+      ref: "slack.post_message"
+      params:
+        channel: "#releasemgmt"
+        message: "```[RELEASE_HAPPY_ENDING]\n STATUS: SUCCESS\n VERSION: {{st2_release_base_ver}}```"
+    -
+      name: "slack_failure"
```
Comment (DoriftoShoes) [inline actions/workflows/st2_package_publish_release.yaml]: I'm not a fan of having the notifications in the action chain.  If the workflows fail I want to see them red in the UI.

File: actions/workflows/st2_package_publish_release.yaml (hunk 3 above)
Comment (manasdk) [inline actions/workflows/st2_package_publish_release.yaml:51]: -a at the end so both packaging jobs run in parallel

File: actions/st2_package_publish_release.meta.yaml (hunk 4 above)
Comment (manasdk) [inline actions/st2_package_publish_release.meta.yaml:2]: following prepare-release ... so your call

File: actions/workflows/st2_package_publish_release.yaml (hunk 5 above)
Comment (manasdk) [inline actions/workflows/st2_package_publish_release.yaml]: Make sense ... 7e9ce2b0be8739187e2c2d1387660658f65eef2b

**Pull request merged**